### ISSUE TITLE
[FR] Option to skip specific DLLs 

### ISSUE THREAD

**Issue by ttodua** [dnSpyEx/dnSpy#97]
Just a feature-request: In Visual Studio, there is option "Just my code", which avoids you navigating through other modules while debugging/stepping-in/out.
Would be good if dnSpyEx also had some option, where we could right click on DLL files in the left sidebar and click "Ignore", so our breakpoints should skip navigating through them while stepping in/out/etc... Because when we check some specific dll, then in many cases we have to navigate through System....dll files, which is frustrating.
Thanks for the work !

**Comment by ElektroKill** [comment]
Hi, would an option to exclude system modules be enough, or do you specifically need to manually choose which modules to exclude from stepping into?

**Comment by ttodua** [comment]
At least, starting from excluding the system modules would have been
useful. then, juat a simple additional field would be excellent, where we
could just list semicolon delimited list of files, like:
ColorPicker.dll;Theme.dll;...
So,if the file name matches, it can skip also.
I hope these way might be most easier to implement. Thanks !

**Comment by ElektroKill** [comment]
Hi, in commit https://github.com/dnSpyEx/dnSpy/commit/bbd89a33322998bcb8291a7a0995cf4631ee2aa9, I added Just My Code support for debugging in dnSpy. This currently comes in the form of three options under the debugger settings page.
![image](https://user-images.githubusercontent.com/37494960/179565952-1008dc52-5257-4607-8dca-23f3b7fee822.png)
The first option enables JMC stepping when debugging and grants access to the two other options. The next option disables stepping into system modules. The last, most extreme, option disabled stepping into any assembly that is not the primary assembly.

I hope that these options will make using the dnSpy debugger easier. As for adding support for user configurable JMC, the current debugger implementation makes this quite challenging. This is why I settled for these two options.
